app/internal/liability/liability.go

```diff
@@ -10,79 +10,68 @@ import (
 )
 
 var (
-	ErrEmptyByteArray       = errors.New("liability: empty byte array")
-	ErrNonExistingLiability = errors.New("liability: there is no such asset in REQ type")
-	ErrReverseOperation     = errors.New("liability: can't revert operation")
-	ErrAcknowledgeOperation = errors.New("liability: acknowledge amount is bigger than actual amount")
+	ErrEmptyByteArray         = errors.New("liability: empty byte array")
+	ErrNonExistingLiabilities = errors.New("liability: liabilities for such participant doesn't exist")
+	ErrNoReqLiability         = errors.New("liability: liability with REQ type doesn't exist")
+	ErrInvalidOperation       = errors.New("liability: given amount is bigger than actual amount")
 )
 
 type Asset string
 
-// Liability represents information about asset and amount of that asset
-type Liability struct {
```

Comment:
let's rename it to liabilitites (plural)
but will have naming for functions like `AddLiability` (with single), cause every time we add only one

Reply:
perfect!